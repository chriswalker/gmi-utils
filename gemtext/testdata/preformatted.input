# Preformatted text

```
This is some preformatted text
```

...and this is not.
