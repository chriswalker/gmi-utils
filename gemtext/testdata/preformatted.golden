  [38;2;0;255;255m# Preformatted text[39m
  
  [38;2;255;0;0mThis is some preformatted text[39m
  
  ...and this is not.
